tests/kafkatest/tests/streams/streams_cooperative_rebalance_upgrade_test.py

```diff
@@ -43,7 +43,10 @@ class StreamsCooperativeRebalanceUpgradeTest(Test):
     second_bounce_phase = "second_bounce_phase-"
 
     # !!CAUTION!!: THIS LIST OF VERSIONS IS FIXED, NO VERSIONS MUST BE ADDED
-    streams_eager_rebalance_upgrade_versions = [str(LATEST_2_1), str(LATEST_2_2), str(LATEST_2_3)]
+    streams_eager_rebalance_upgrade_from_versions = [str(LATEST_2_1), str(LATEST_2_2), str(LATEST_2_3)]
+
+    # !!CAUTION!!: THIS LIST OF VERSIONS IS FIXED, NO VERSIONS MUST BE ADDED
+    streams_cooperative_rebalance_upgrade_to_versions = [str(LATEST_2_4), str(LATEST_2_5), str(LATEST_2_6), str(LATEST_2_7), str(LATEST_2_8), str(LATEST_3_0), str(LATEST_3_1), str(LATEST_3_2), str(LATEST_3_3), str(LATEST_3_4), str(LATEST_3_5), str(LATEST_3_6), str(LATEST_3_7), str(LATEST_3_8), str(LATEST_3_9)]
```

Comment:
on second thought, should we just go ahead and remove this test in 4.0? if it's only testing up to earlier versions then it's not doing much here

Reply:
Yes, makes sense. We can drop the test entirely

Reply:
I believe we can also drop `StreamsUpgradeToCooperativeRebalanceTest.java` when dropping this system test? Not sure if there is other Java classes that are only used by this test that we could drop?